### ISSUE TITLE
Delete blobs associated with blocked users

### ISSUE THREAD

**Issue by KyleMaas** [arj03/ssb-browser-demo#74]
I'm not sure how you'd do this considering blob storage seems to have no metadata associated with blobs, but it would be nice if, when a profile is blocked and their messages deleted, we could clear out blobs which came from that person as well, just to make sure that if the user had a blob we don't want that it doesn't just stick around forever.

Related to #68.

**Comment by arj03** [comment]
Yes, it seems the blob purge modules knows which feed a blob might belong to.

**Comment by KyleMaas** [comment]
The lack of blob metadata is one of the things I was struggling with for ssb-blobs-purge - in that case, not having a timestamp.

**Comment by KyleMaas** [comment]
As it is, I'm not sure this one will be possible, for two reasons:

1. We really have no good way to list all of the blobs in storage.  This is a limitation of RAF, which gives us no way to do that, so our "best guess" way to do it is too look at all the blobs we *might* have and check and see if we *do*.
2. Blobs have no ownership information stored with them, or really any other information.  We don't know who they came from, we don't know which post pulled them into our storage, and we don't know when they were stored.
